docs/csharp/programming-guide/strings/index.md

```diff
@@ -118,7 +118,7 @@ string s = String.Empty;
   
 |Topic|Description|  
 |-----------|-----------------|  
-|[How to: Modify String Contents](../../../csharp/programming-guide/strings/how-to-modify-string-contents.md)|Provides a code example that illustrates how to modify the contents of strings.|  
+|[How to: Modify String Contents](../../how-to/modify-string-contents.md)|Provides a code example that illustrates how to modify the contents of strings.|  
```

Comment:
Instead of *Provides a code example...* it better sounds like in a row below on concatenation: *Illustrates...* Not for this PR: description of *How to: Parse Strings...* also might be updated.
Though, description of concatenation is not correct too: string methods are also used for concatenation. I'd review that table as a whole.